dom/dom_c.php

```diff
@@ -1328,8 +1331,10 @@ public function item(#[LanguageLevelTypeAware(['8.0' => 'int'], default: '')] $i
      */
     #[TentativeType]
     public function getNamedItemNS(
-        #[LanguageLevelTypeAware(['8.0' => 'string|null'], default: '')] $namespace,
-        #[LanguageLevelTypeAware(['8.0' => 'string'], default: '')] $localName
+        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $namespaceURI = '',
```

Comment:
`$namespaceURI` is available till 7.4: https://3v4l.org/mRrva#v700, without default value.
`$namespace` is available only since 8.0.
`$localName` doesn't have a default value?: https://3v4l.org/em9Kr#v540

Reply:
Updated to
```
#[TentativeType]
    public function getNamedItemNS(
        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.4')] $namespaceURI = '',
        #[PhpStormStubsElementAvailable(from: '8.0')] ?string $namespace,
        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.4')] $localName = '',
        #[PhpStormStubsElementAvailable(from: '8.0')] string $localName
    ): ?DOMNode {}
```